### ISSUE TITLE
REALMS expansion

### ISSUE THREAD

**Issue by brazilofmux** [brazilofmux/tinymux#98]
Original [issue 95](https://code.google.com/p/tinymux/issues/detail?id=95) created by brazilofmux on 2006-09-15T16:47:12.000Z:

My understanding of the REALMS codes is that it compares values for
Obf_level and hss_level. If one is higher, that wins, if they tie, then
it's a 50/50 random chance.

What I'd like to see is an expansion of these attributes so that game
admins could add softcode and have it evaluated to return a number which
could vary based on the looker/lookee's various chargen statistics. In this
way you would better be able to handle the diverse methods of sensing, and
hiding from sight through softcode.

Basically the system needs an integer value to make the comparison, so,
like a softcode lock, it would evaluate a function in the attribute which
would return an integer value, or if it didn't return an integer, would
return 0. If all that's there is an integer, then no evaluation occurs.

This would allow softcode development to expand REALMS without affecting
backwards compatability.

DK
